src/lib/mina/actions/offchain-contract-tests/multi-contract-instance.unit-test.ts

```diff
@@ -0,0 +1,184 @@
+import {
+  SmartContract,
+  method,
+  state,
+  PublicKey,
+  UInt64,
+} from '../../../../index.js';
+import * as Mina from '../../mina.js';
+import assert from 'assert';
+import { ExampleContract } from './ExampleContract.js';
+import { settle, transfer } from './utils.js';
+
+const Local = await Mina.LocalBlockchain({ proofsEnabled: false });
```

Comment:
proofs not enabled on the multi-contract test since it takes much longer.  This also exercises both the proofs-enabled and proofs-disabled paths.